source/administration/changelog.md

```diff
@@ -37,9 +35,10 @@ Added significant improvements for large organizations running on a single serve
 #### Favorite Channels
 - Added the ability to select Favorite Channels that appear at the top of the channels sidebar.
 
-#### Video and Audio Calling (Beta)
+#### Video and Audio Calling (early-preview)
 - Added early preview of video and audio calling option using self-hosted proxy.
 - Intended as working prototype for community development, not recommended for production.
+- Early preview does not include logging or detailed documentation. 
```

Comment:
Is this line necessary? There's going to be docs letting people know how to set up WebRTC (the PR is already merged against 3.5 branch).